rest_api/sawtooth_rest_api/route_handlers.py

```diff
@@ -580,19 +580,10 @@ def __init__(
                                payload, error_traps=None):
         """Sends a request to the validator and parses the response.
         """
-        LOGGER.debug(
```

Comment:
I don't think these should be removed, since they are valid log messages.  I would wrap them in an `if log_level == DEBUG` (or how ever that needs to be done), such that the expensive calls are not done on info or higher.

Reply:
Also, I'd pull this commit out all together and do it in a separate PR